packages/elements/src/__fixtures__/services/without-origin.ts

```diff
@@ -0,0 +1,69 @@
+import { IHttpService } from '@stoplight/types';
+
+export const httpServiceWithoutOrigin: IHttpService = {
+  id: '?http-service?',
+  name: 'Petstore (OpenAPI v3)',
```

Comment:
Can we keep petstore usage to a minimum? It's a nit of a running joke in the OAS community, and... not a good joke.

Reply:
Honestly I just copied some other fixture and modified it for the specific test.

Do you want me to change it?